packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -134,6 +142,7 @@ class ViewportPositioner extends Foundation<
         verticalAlwaysInView: void 0,
         verticalLockToDefault: void 0,
         fixedAfterInitialPlacement: void 0,
+        scaleToAvailableSpace: void 0,
```

Comment:
I'd recommend `scaleToFit`, or `autoResize` as more common naming for this action.

Reply:
"ScaleToFit" I like